packages/wonder-blocks-tokens/src/tokens/media-queries.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * A default set of media queries to use for different screen sizes.
+ *
+ * Breakpoint documentation: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2099970518/Layout+Breakpoints
+ *
+ */
+
+export const pureWidths = {
+    xsMax: 567,
+    smMin: 568,
+    smMax: 681,
+    mdMin: 682,
+    mdMax: 1023,
+    lgMin: 1024,
+} as const;
```

Comment:
**suggestion:** I feel that the `pureWidths` naming is too open. It would be better if we could give it a name that relates to media queries / breakpoints.

Related to your comment about media queries vs breakpoints, how do you feel about exporting this as an object that contains media query breakpoints and width breakpoints (or something like that)?

e.g.
```
export const breakpoint = {
  width: pureWidths,
  mediaQuery (or mq): the object with all the mq breakpoints
};
```

We use this approach with other tokens already, so this could bring some consistency to the token naming structure.

Reply:
Works for me, I love objects! 😂